src/main.rs

```diff
@@ -109,13 +109,24 @@ fn process_result(req_id: String) {
     }
 }
 
+fn make_config() -> Result<Config> {
+    let mut config = Config::default();
+
+    //TODO-neel add shutdown_timeout later
```

Comment:
```suggestion
    // TODO(neel): add shutdown_timeout later
```

Not sure if all tools support your format, but afaik the one proposed is more common :P